File: SDKs/Aspose.Pdf_Cloud_SDK_for_CSharp/src/Com/Aspose/PDF/Model/TextReplace.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.PDF.Model {
  public class TextReplace {
    public string OldValue { get; set; }

    public string NewValue { get; set; }

    public bool? Regex { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class TextReplace {\n");
      sb.Append("  OldValue: ").Append(OldValue).Append("\n");
      sb.Append("  NewValue: ").Append(NewValue).Append("\n");
      sb.Append("  Regex: ").Append(Regex).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }

File: SDKs/Aspose.Pdf-Cloud-SDK-for-Java/src/main/java/com/aspose/pdf/model/TextReplace.java
package com.aspose.pdf.model;

public class TextReplace {
  private String OldValue = null;
  private String NewValue = null;
  private Boolean Regex = null;
  /**
	 * getOldValue
	 * Gets String
	 * @return OldValue
	 */
  public String getOldValue() {
    return OldValue;
  }

	/**
	 * setOldValue
	 * Sets String
	 * @param OldValue String
	 */
  public void setOldValue(String OldValue) {
    this.OldValue = OldValue;
  }

  /**
	 * getNewValue
	 * Gets String
	 * @return NewValue
	 */
  public String getNewValue() {
    return NewValue;
  }

	/**
	 * setNewValue
	 * Sets String
	 * @param NewValue String
	 */
  public void setNewValue(String NewValue) {
    this.NewValue = NewValue;
  }

  /**
	 * getRegex
	 * Gets Boolean
	 * @return Regex
	 */
  public Boolean getRegex() {
    return Regex;
  }

	/**
	 * setRegex
	 * Sets Boolean
	 * @param Regex Boolean
	 */
  public void setRegex(Boolean Regex) {
    this.Regex = Regex;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class TextReplace {\n");
    sb.append("  OldValue: ").append(OldValue).append("\n");
    sb.append("  NewValue: ").append(NewValue).append("\n");
    sb.append("  Regex: ").append(Regex).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

